src/components/templates/HackathonProjects.js

```diff
@@ -18,10 +27,11 @@ import { useSubmissions } from "../../hooks";
 import Spinner from "../molecules/Spinner";
 
 const HackathonProjects = () => {
-  const { id } = useParams();
+  const id = useParams().id || 1;
```

Comment:
I can't tell you how happy this one little line makes me, just knowing that it works